ipatests/test_ipaserver/test_adtrust_mockup.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2018  FreeIPA Project Contributors - see LICENSE file
+
+from __future__ import print_function
+import ipaserver.install.adtrust as adtr
+from ipaserver.install.adtrust import set_and_check_netbios_name
+from collections import namedtuple
+from unittest import TestCase
+from unittest.mock import MagicMock
```

Comment:
FYI, this only works on Python 3. For Python 2.7, you have to use the external package ``mock``.